addons/website_sale/static/src/js/website_sale_utils.js

```diff
@@ -69,8 +95,16 @@ function updateCartNavBar(data) {
         }
     }
 
-    $(".js_cart_lines").first().before(data['website_sale.cart_lines']).end().remove();
-    $("#cart_total").replaceWith(data['website_sale.total']);
+    const jsCartLinesEl = document.querySelector(".js_cart_lines");
```

Comment:
I know it's a weird design but, at the moment, there can be multiple `. js_cart_lines` elements (`warning` and lines). Both of them should be replaced by the server call. Please check `odoo/addons/website_sale/views/templates.xml` for more information.
```suggestion
    const jsCartLinesEl = document.querySelectorAll(".js_cart_lines");
```